### PR TITLE
Update code to match Ruby styleguide

### PR DESCRIPTION
**Pull request by dlitvakb**

### CODE REVIEW

File: lib/contentful_model/associations/belongs_to.rb [hunk 1]
```diff
@@ -1,17 +1,22 @@
 module ContentfulModel
   module Associations
+    # Defines Belongs To association
     module BelongsTo
       def self.included(base)
         base.extend ClassMethods
       end
 
+      # Class method
```
Comment (mariobodemann) [inline lib/contentful_model/associations/belongs_to.rb]: Hmm, that is a comment of limited use. Is there a way to make it more expressive/informative? Similar discussion could be started about the `Defines Belongs To association`: is there something we can use to add more value to the comments?

File: lib/contentful_model/version.rb [hunk 2]
```diff
@@ -1,3 +1,3 @@
 module ContentfulModel
-  VERSION = "0.2.0"
+  VERSION = '0.2.0'.freeze
```
Comment (mariobodemann) [inline lib/contentful_model/version.rb]: Shouldn't the version number increase? Since you are renaming something, I was hoping for a change in here too.

**Comment by andyhansen** [comment]
Hey there, the removal of `AttributeNotFoundError` has broken some of our code which relied on it. In this case, should we be using `NoMethodError` instead?

**Comment by dlitvakb** [comment]
Hey @andyhansen,

Yes, that would be the best solution. I found that there was no benefit internally for wrapping the error as it should be handled in the same way to avoid strange behaviour and some edge cases (there used to be 2 handlers internally, one for `NoMethodError` and one for `AttributeNotFoundError` for every single exception check, which made no sense in many cases.

Otherwise, I'd advise to use `return_nil_for_empty` in the cases in which this is an issue for you.

Hope this explains the reasoning a bit further,

Cheers

**Pull request merged**